### PR TITLE
Add dconf user config to fix terminal font display

### PR DESCRIPTION
**Pull request by manics**
I've no idea why but it's been like this for ages:
image

After this fix
image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -33,3 +33,11 @@ COPY --chown=$NB_UID:$NB_GID . /opt/install
...
     pip install -e /opt/install && \
     jupyter server extension enable jupyter_remote_desktop_proxy
+
+RUN mkdir -p /home/$NB_USER/.config/dconf/
+# The default terminal font displays poorly in XFCE4
+# The dconf CLI requires the dconf-service so can't be run at build time
+# To recreate this file run:
+#   dconf write /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/font "'Liberation Mono 12'"
+#   dconf write /org/gnome/terminal/legacy/profiles:/:b1dcc9dd-5262-4d8d-a863-c897e6d979b9/use-system-font false
+COPY --chown=$NB_UID:$NB_GID dconf/user /home/$NB_USER/.config/dconf
```
Comment (consideRatio) [inline Dockerfile:43]: Can this be put where it won't be replaced by a user home directory mount instead - under etc or similar?

**Review by consideRatio — COMMENTED** [review]
I'd appreciate getting a better separation of things for the Dockerfile from whats for the python package etc. Not sure on what i propose, but putting a binary blob file  in a dconf/ folder will make this repo a bit harder to understand by glancing on files in it.

(This isn't a comment that needs to be addressed)

**Comment by yuvipanda** [comment]
Testing https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/86 as an alternative fix

**Comment by yuvipanda** [comment]
#86 fixes it! 

image

**Pull request closed without merging**